src/api/app/controllers/webui/package_controller.rb

```diff
@@ -1034,6 +1034,15 @@ def save_meta
   def edit
   end
 
+  def jobshistory
```

Comment:
This could go in its own controller so we dont add more and more stuff to PackageController which is already > 1000 lines long. For example the controller and action could be `Webui::Packages::JobHistoryController#index`